study.md

```diff
@@ -21,7 +21,7 @@ material. Instead, digest what you've read and repeat it in your own voice.
 Write the Ruby code that takes the integer `700` and returns the string `"007"`.
 
 ```ruby
-# your answer here
+"700".reverse #=>
```

Comment:
This reverses the string "700" and returns the correct value, but the instructions were to start with an integer.  `700.to_s.reverse`.